### ISSUE TITLE
Web Notification Not Working (Chrome)

### ISSUE THREAD

**Issue by ranman09** [humhub/humhub#3220]
At a notification, a message or a comment, no web notification is seen

### What is the expected result?
A notification to alert each user

### What do you get instead?

Intead I get a nothing.
### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.2.8
| PHP version      | 
| Operating system |

> Note: Please use the [community](https://community.humhub.com) site for any questions on customizing HumHub.
> Note: Also provide the full error stacks from `Administration --> Information --> Logging` and errors from your browser console if possible.

**Comment by buddh4** [comment]
You mean if the browser is closed or even if its open with a humhub tab open?

**Comment by ranman09** [comment]
Even if its open or minimized. Doesn't have any notification. It didn't even ask for me to allow notifications (the prompt like at page load)

**Comment by buddh4** [comment]
Could not recreate this did you activate this in your notification settings 'Account Settings -> Notificaitons -> Receive desktop notifications when you are online.' ?

**Comment by Felli** [comment]
I can confirm this does happen for Linux CentOS 6 running localhost.

**Comment by ranman09** [comment]
@buddh4 yes I can confirm I have activated every notification I can activate.

@felli I will try to confirm my OS. I deployed it on a web hosting.
